iot-e2e-tests/android/app/build.gradle

```diff
@@ -14,7 +14,7 @@ def IS_BASIC_TIER_HUB = project.hasProperty('IS_BASIC_TIER_HUB') ? '"'+project.p
 def IS_PULL_REQUEST = project.hasProperty('IS_PULL_REQUEST') ? '"'+project.property('IS_PULL_REQUEST')+'"' : '"false"'
 def IOTHUB_CLIENT_ID = project.hasProperty('IOTHUB_CLIENT_ID') ? '"'+project.property('IOTHUB_CLIENT_ID')+'"' : '""'
 def IOTHUB_CLIENT_SECRET = project.hasProperty('IOTHUB_CLIENT_SECRET') ? '"'+project.property('IOTHUB_CLIENT_SECRET')+'"' : '""'
```

Comment:
@vinagesh it turns out I already had the "IOTHUB_" prefix for these environment variables, so I just needed to change the TENANT_ID to be prefixed with MSFT